src/parser/jobs/blm/RotationWatchdog.js

```diff
@@ -8,16 +8,15 @@ import {ActionLink} from 'components/ui/DbLink'
 import Rotation from 'components/ui/Rotation'
 import ACTIONS, {getAction} from 'data/ACTIONS'
 import Module from 'parser/core/Module'
-import {FIRE_SPELLS, ICE_SPELLS} from 'parser/jobs/blm/Elements'
 import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
 
 const EXPECTED_FIRE4 = 6
 const FIRE4_FROM_CONVERT = 2
 
 const DEBUG_LOG_ALL_FIRE_COUNTS = false
 
-export default class FireCounter extends Module {
-	static handle = 'firecounter'
+export default class rotation extends Module {
```

Comment:
Please name your classes the same as your File, as well as use Pascal case for classes. => `RotationWatchdog`